src/javascript/app_2/App/Components/Form/TimePicker/dialog.jsx

```diff
@@ -85,12 +85,12 @@ Dialog.propTypes = {
     ]),
     onChange  : PropTypes.func,
     preClass  : PropTypes.string,
-    start_time: PropTypes.number,
-    value     : PropTypes.oneOfType([
+    start_time: PropTypes.oneOfType([
         PropTypes.number,
         PropTypes.string,
         PropTypes.object,
     ]),
+    value: PropTypes.string,
```

Comment:
was the change of proptype intentional? value can only be string, and start time can be number, string, or object?

Reply:
Yes, the changes previously was a fault. It was meant for start time to have those. Please look at the commit message.